content/ita/careers.md

```diff
@@ -12,54 +12,17 @@ twitterlargeimage: "/images/careers/banner.png"
 
 # Cosa offriamo in Facile.it:
 
  * Assunzione a tempo indeterminato con RAL in base alla tua esperienza + fondo salute integrativo
  * Possibilità di crescita economica in base alle nuove competenze acquisite
  * Un venerdì al mese dedicato alla formazione e al team building
  * Biblioteca di libri di tecnologia con più di 100 titoli
- * Budget annuale di 1.500€ per l’hardware e software di cui hai bisogno
- * Partecipazione ad eventi di tecnologia a nostro carico (Pragma Conference, Droidcon, Codemotion, etc)
+ * Partecipazione ad eventi di tecnologia a nostro carico (SymfonyCon, phpDay, Codemotion, etc.)
  * Ufficio in centro a Milano (MM3 Missori) con sala relax, sofà, ping-pong e calcetto
- * Convenzioni (bar, ristoranti, palestre and more!), Ticket Restaurant, caffè, acqua e frutta fresca ogni giorno
+ * Convenzioni (bar, ristoranti, palestre e molto altro), Ticket Restaurant, caffè, acqua e frutta fresca ogni giorno
  * Fare il lavoro che ami in una realtà italiana di successo con un top team al tuo fianco
 
-<a href="http://jobs.facile.it/chi-cerchiamo/candidati.html" target="\_blank">Candidati »</a>
-
-Quasi tutte le nostre applicazioni si basano sulla piattaforma LAMP. Per entrare nel nostro team devi quindi conoscere bene strumenti quali Linux, Apache, MySQL e PHP, oltre a rispondere “sì” a queste domande:
-
-# Candidato base
-
- * Ti appassiona la programmazione, in particolare la programmazione a oggetti?
- * Hai mai lavorato con web service?
- * Hai familiarità con librerie JS come jQuery?
- * Ti senti a tuo agio nell’interrogare e progettare database?
- * Ami lavorare in team e condividere quello che impari?
-
-**È un valore aggiunto se...**
-
- * conosci i principali Design Pattern;
- * usi sistemi di continuous integration;
- * hai esperienza con test automatici;
- * sei appassionato di Open Source.
-
-# Candidato senior
- 
- * La programmazione a oggetti è il tuo pane quotidiano?
- * Hai almeno cinque anni di esperienza come sviluppatore web?
- * Utilizzi un sistema di versionamento del codice (preferibilmente Git)?
- * Sai cosa sono i design pattern, le regole del SOLID e li utilizzi?
- * Conosci i principi del REST?
- * Hai sviluppato delle API REST e del codice per consumarle?
- * Ami lavorare in team e condividere quello che impari?
- * Ti interessa progredire nelle tue abilità tecniche, studiare e migliorarti costantemente?
-
-**È un valore aggiunto se...**
-
- * hai esperienza approfondita con uno dei principali framework (Symfony 2 / 3, Zend 2 / 3, etc);
- * hai esperienza approfondita con test automatici e TDD;
- * usi sistemi di continuous integration;
- * hai un account su GitHub e hai dimestichezza col software Open Source;
- * ti interessa lavorare con altre tecnologie quali Docker, RabbitMQ, MongoDB.
+<a href="https://jobs.facile.it/chi-cerchiamo.html" target="\_blank">Posizioni aperte »</a>
```

Comment:
```suggestion
Se sei interessato, dai un occhio alle nostre <a href="https://jobs.facile.it/chi-cerchiamo.html" target="\_blank">offerte di lavoro</a>.
```

Reply:
"Se sei interessato, dai uno sguardo alle nostre offerte di lavoro" is it ok?